website/source/docs/installation.html.md

```diff
@@ -28,15 +28,17 @@ installed. On Unix systems, `~/packer` or `/usr/local/packer` is generally good,
 depending on whether you want to restrict the install to just your user or
 install it system-wide. On Windows systems, you can put it wherever you'd like.
 
-After unzipping the package, the directory should contain a set of binary
-programs, such as `packer`, `packer-build-amazon-ebs`, etc. The final step to
+After unzipping the package, the directory should contain a single binary
+called `packer`. The final step to
```

Comment:
My non-native English would add a _the_ here so it would read:
> The final step to _the_ installation is to make sure the directory you installed Packer to is on the PATH.